client/tx/factory.go

```diff
@@ -471,14 +474,19 @@ func (f Factory) Prepare(clientCtx client.Context) (Factory, error) {
 	}
 
 	initNum, initSeq := fc.accountNumber, fc.sequence
-	if initNum == 0 && initSeq == 0 {
+	if initNum == 0 || initSeq == 0 {
```

Comment:
Actually, we really need this condition, as we may want to purposely set the sequence to 0 in offline mode and this should not try to be populated.